### DISCUSSION TITLE
HA e bluetooth le

### DISCUSSION THREAD

**Discussion by orgnemo** [indomus/forum#196] [GENERALE - Discussioni generiche]
Salve a tutti,
mi devono installare la pompa di calore al posto della caldaia, volevo cogliere l'occasione per mettere dei sensori temperatura in tutte le stanze in modo da modulare il calore nelle singole stanze.  Adesso usavo un esp01 con sensore di temperatura, solo nelle camere, ma ho visto gli  Xiaomi Mijia  che sono molto carini esteticamente e danno più informazioni, inoltre si possono flashare con ESPhome, il problema è che il mio rasberry è nella sala tecnica e dalle prove che ho fatto il segnale bluetooth arriva solo in solone.

Ho pensato di fare un gateway con gli esp8266 e poi aggiornare via mqtt o api, volevo sapere se ci sono soluzioni più semplici migliori di questa... mantenendo una spesa contenuta, voi come fate?

Grazie

**Comment by snakuzzo** [comment]
La soluzione che hai indicato credo sia la più conveniente...in alternativa...butta quei sensori e compra qualcosa non BT ma magari Zigbee :)

**Reply by orgnemo** [reply]
Con Zigbee ho ugualmente il problema della distanza? Se si come lo posso gestirlo? Quali sono le soluzioni più adottate?

Grazie

**Reply by snakuzzo** [reply]
Il problema della distanza si può risolvere utilizzando poi appositi device che fungono anche da router. I dispositivi Zigbee sono in mesh. Leggi [QUI](https://indomus.it/componenti/zigbee/). Ad ogni modo puoi anche scegliere un coordinator che abbia una "bella portata".
Quanto alle soluzioni relative al coordinator...hai diverse scelte. Dai un'occhiata anche [QUI](https://indomus.it/componenti/z-bee-duo-zigbee-coordinator-usb-hat-per-raspberry-pi-zero/)

**Reply by orgnemo** [reply]
Grazie  ho molti spunti

**Comment by 50494554524F** [comment]
se hai uno di questi

https://github.com/pvvx/ATC_MiThermometer

puoi modificar la potenza dell'antenna... lo sviluppatore, riassumendo, dice che la piattaforma esp non è buona con bluetooth perché perde tutti i pacchetti, da verificare

**Reply by Olodumdomotico** [reply]
Sicuramente più costoso ma completo e duttile c'è il termostato evohome adatto anche per riscaldamento a pavimento o espandibile con valvole termostatiche nei radiatori

**Reply by orgnemo** [reply]
Grazie in effetti il passo successivo sarebbero state le valvole termostatiche, ma il prezzo è alto.
